scripts/apply_def_template.py

```diff
@@ -55,21 +56,30 @@ def remove_power_pins(DEF):
     templateDEF = f"{userDEF}.template.tmp"
     remove_power_pins(templateDEF)
 
-    subprocess.check_output(
-        [
-            "openroad",
-            "-python",
-            f"{scriptsDir}/defutil.py",
-            "replace_pins",
-            "--output",
-            userDEF,
-            "--input-lef",
-            "/dev/null",
-            userDEF,
-            templateDEF,
-        ],
-        stderr=subprocess.PIPE,
-    )
+    shutil.copy(userDEF, f"{userDEF}.user.template.tmp")
+
+    try:
+        print(
```

Comment:
Why not just `subprocess.check_call` if you want to spit out the output anyway?

Additionally, defutil.py is not supposed to generate much in terms of output…

Reply:
Will replace with check_call.

I think we should output the log, as this implementation, while solving def pin locations issue, will crash when strange input is supplied (like broken def). The log will help us debug this kind of issues. But I think, I will add an option to write it to some logfile instead of stdout.

Reply:
Makes sense.